### PR TITLE
Adapt scripts to upcoming RFC version

### PR DESCRIPTION
**Pull request by vbnogueira**

### CODE REVIEW

File: calcPNA/calc.parser [hunk 1]
```diff
@@ -14,6 +14,12 @@ die()
 }
 
 IP="ip"
+#md_off is (hdrfieldid - 1) * 2
+id2mdoff()
+{
+	echo $((($1 - 1)*2))
+}
```
Comment (jhsmt) [inline calcPNA/calc.parser:21]: just call it headerid - it is more readable that way..

File: test_003/p4tc_test_003.purge [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -x
+
+TC="/mnt/p4tc-iproute2/tc/tc"
```
Comment (jhsmt) [inline test_003/p4tc_test_003.purge:3]: Seems to be a leftover

File: stateful-tcp-flow-collector/stateful1/stateful1.purge [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+TC="tc"
+
+set -e
+
+DEV=port0
+
+$TC qdisc del dev $DEV clsact
+$TC qdisc del dev $DEV root
+sleep 1
```
Comment (jhsmt) [inline stateful-tcp-flow-collector/stateful1/stateful1.purge:11]: Do we still need root here if we have ingress vs egress filters on clsact?

**Review by jhsmt — CHANGES_REQUESTED** [review]

File: calcPNA/calc.parser (hunk 1 above)
Comment (vbnogueira) [reply calcPNA/calc.parser:21]: Done here: https://github.com/p4tc-dev/p4tc-examples/pull/1/files#diff-06c1942853ecafac2746798b52962865001a03750589f7d81845a2a9af263c1fR18

File: stateful-tcp-flow-collector/stateful1/stateful1.purge (hunk 3 above)
Comment (vbnogueira) [reply stateful-tcp-flow-collector/stateful1/stateful1.purge:18]: Done here: https://github.com/p4tc-dev/p4tc-examples/pull/1/files#diff-9623f2fc86b847168a124d9382aff2d92fcdfe24c5bf3ce7dab679564361c845R17

File: test_003/p4tc_test_003.purge (hunk 2 above)
Comment (vbnogueira) [reply test_003/p4tc_test_003.purge]: Removed here: https://github.com/p4tc-dev/p4tc-examples/pull/1/files#diff-cf5ad57aee13e201c6914e7af985290b6f6543941a1216b7ec3c237491e0d7afR3

**Review by jhsmt — APPROVED** [review]

**Pull request merged**